mmv1/third_party/validator/bigtable_cluster.go

```diff
@@ -0,0 +1,105 @@
+package google
+
+import (
+	"reflect"
+)
+
+func GetBigtableClusterCaiObject(d TerraformResourceData, config *Config) ([]Asset, error) {
+	
+	objs, err := GetBigtableClusterApiObjects(d, config); 
+	
+	if err != nil {
+		return []Asset{}, err
+	}
+
+    assets := []Asset{}
+	for _, obj := range objs {
+		name, err := assetName(d, config, "//bigtable.googleapis.com/projects/{{.Provider.project}}/instances/{{name}}/clusters/{{cluster_id}}")
+		if err != nil {
+			return []Asset{}, err
+		}
+
+		asset := Asset {
+			Name: name,
+			Type: "bigtableadmin.googleapis.com/Cluster",
+			Resource: &AssetResource{
+				Version:              "v2",
+				DiscoveryDocumentURI: "https://bigtableadmin.googleapis.com/$discovery/rest",
+				DiscoveryName:        "Cluster",
+				Data:                 obj,
+			},
+		}
+		assets = append(assets, asset)
+	}
+	return assets, nil
+}
+
+func GetBigtableClusterApiObjects(d TerraformResourceData, config *Config) ([]map[string]interface{}, error) {
+	return expandBigtablClusters(d.Get("cluster"), d, config)
+	
+}
+
+func expandBigtablClusters(v interface{}, d TerraformResourceData, config *Config) ([]map[string]interface{}, error) {
```

Comment:
```suggestion
func expandBigtableClusters(v interface{}, d TerraformResourceData, config *Config) ([]map[string]interface{}, error) {
```

Reply:
All good mate. Thanks for picking up the typo ;)